diff -up dnfdragora-1.1.2/dnfdragora/backend.py.2~ dnfdragora-1.1.2/dnfdragora/backend.py
--- dnfdragora-1.1.2/dnfdragora/backend.py.2~	2019-06-14 20:04:24.118852311 +0200
+++ dnfdragora-1.1.2/dnfdragora/backend.py	2019-06-14 20:04:56.036852635 +0200
@@ -135,7 +135,7 @@ class ArchFilter(BaseFilter):
 
     def __init__(self, name, active=False):
         BaseFilter.__init__(self, name, active)
-        self.archs = ['noarch', 'i586', 'i686', 'x86_64', 'armv5tl', 'armv6hl', 'armv7hl', 'aarch64', 'armv7hnl', 'armv8hnl', 'aarch64', 'ppc64', 'ppc64le', 's390x']
+        self.archs = ['noarch', 'i586', 'i686', 'znver1_32', 'x86_64', 'znver1', 'armv5tl', 'armv6hl', 'armv7hl', 'aarch64', 'armv7hnl', 'armv8hnl', 'aarch64', 'ppc64', 'ppc64le', 's390x']
 
     def run(self, pkgs):
         BaseFilter.run(self, pkgs)
diff -up dnfdragora-1.1.2/dnfdragora/const.py.2~ dnfdragora-1.1.2/dnfdragora/const.py
--- dnfdragora-1.1.2/dnfdragora/const.py.2~	2019-06-14 20:04:24.118852311 +0200
+++ dnfdragora-1.1.2/dnfdragora/const.py	2019-06-14 20:05:40.533853087 +0200
@@ -44,8 +44,8 @@ ARCH = subprocess.check_output(
     '/usr/bin/rpm --eval %_arch', shell=True).decode("utf-8")[:-1]
 
 ARCH_DICT = {
-    "x86_64": set(['x86_64', 'i686', 'i586', 'i386', 'noarch']),
-    "i386": set(['i686', 'i586', 'i386', 'noarch']),
+    "x86_64": set(['znver1', 'x86_64', 'znver1_32', 'i686', 'i586', 'i386', 'noarch']),
+    "i386": set(['znver1_32', 'i686', 'i586', 'i386', 'noarch']),
     "armhfp": set(['armv7hl', 'armv7hnl', 'armv8hnl', 'noarch']),
     "aarch64": set(['aarch64', 'noarch']),
     "ppc64le": set(['ppc64le', 'noarch']),
