diff --git a/dnfdragora/groupicons.py b/dnfdragora/groupicons.py
index ab2b3c4..476b127 100644
--- a/dnfdragora/groupicons.py
+++ b/dnfdragora/groupicons.py
@@ -6,10 +6,10 @@ class GroupIcons:
 
         if icon_path:
             self.icon_path = icon_path if icon_path.endswith('/') else icon_path + "/"
-            self.mini_icon_path = self.icon_path + "mini/"
+            self.mini_icon_path = self.icon_path + "mini_/"
         else:
             self.icon_path = '/usr/share/icons/'
-            self.mini_icon_path = '/usr/share/icons/mini/'
+            self.mini_icon_path = '/usr/share/icons/mini_/'
 
         # TODO add a localized string for any group in "title"
         self._group_info = {
