diff --git a/dnfdragora/ui.py b/dnfdragora/ui.py
index 37bbcd0..49c904f 100644
--- a/dnfdragora/ui.py
+++ b/dnfdragora/ui.py
@@ -235,6 +235,14 @@ class mainGui(dnfdragora.basedragora.BaseDragora):
               misc.logger_setup(log_filename, loglvl=logging.DEBUG)
             else:
               misc.logger_setup(log_filename)
+            try:
+                if self.level_debug:
+                    misc.logger_setup(self.log_filename, loglvl=logging.DEBUG)
+                else:
+                    misc.logger_setup(self.log_filename)
+            except PermissionError as e:
+                print(_("Can't write to log file %s, logging to stderr instead") % self.log_filename, file=sys.stderr)
+                self.log_filename = None
             print("Logging into %s, debug mode is %s"%(self.log_directory, ("enabled" if self.level_debug else "disabled")))
             logger.info("dnfdragora started")
         else:
diff --git a/etc/dnfdragora.yaml.in b/etc/dnfdragora.yaml.in
index e2fab2e..18cbdd4 100644
--- a/etc/dnfdragora.yaml.in
+++ b/etc/dnfdragora.yaml.in
@@ -2,8 +2,8 @@ settings:
     use_comps: @USE_COMPS@
     always_yes: False
     update_interval: 180
-#    log_filename: PATH/TO/dnfdragora.log
-#    log_level_debug: True
+    log_filename: /var/log/dnfdragora.log
+    log_level_debug: True
 
 #    path:
 #        group_icons: /usr/share/icons
